src/GraphQL.Tests/Bugs/Issue1189.cs

```diff
@@ -30,6 +30,7 @@ public class Issue1189 : SchemaBuilderTestBase
         public void Issue1189_Should_Work(Type resolverType, string errorMessage, string code)
         {
             Builder.Types.Include<Issue1189_Query>();
+            Builder.Types.For("Character").Type = typeof(Issue1189_Character);
```

Comment:
Ping @lanwin . This is a good example of the case you described.

Reply:
Yes it seems so. But my wish would be that I know that this is wrong before someone issues the first query against the api.

Reply:
Of course there will be an error while building the schema.